docs/dotnetcore/introducing-dotnetcore/introducing-dotnetcore.rst

```diff
@@ -0,0 +1,94 @@
+Introducing .NET Core
+=====================
+By `Steve Smith`_ | Originally Published: 1 June 2015 
+
+.. _`Steve Smith`: Author_
+
+.NET Core is a small, optimized runtime that can be targeted by ASP.NET 5 applications. In fact, the new ASP.NET 5 project templates target .NET Core by default, in addition to the full .NET Framework. Learn what targeting .NET Core, or *dnxcore50*, means for your ASP.NET 5 application.
```

Comment:
Removed "full". Since this is the intro, I don't want to get into what dnxcore50 is here, but it is mentioned below (and can be elaborated upon if you think that's necessary).